docker-compose.yml

```diff
@@ -1,6 +1,13 @@
 version: "3"
 
 services:
+  mongodb:
```

Comment:
If you're expecting users to always use this local mongo when running in docker compose, it would be nice to provide/override their database url environment variable; otherwise locally I was spinning up this mongo database but not using it!